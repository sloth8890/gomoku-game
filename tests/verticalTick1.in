place A1
place P1
place A2
place S1
place A3
place H1
place A4
place H2
place A5
