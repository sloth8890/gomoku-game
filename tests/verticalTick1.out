Black wins!
A1P1A2S1A3H1A4H2A5
Thank you for playing!